### ISSUE TITLE
Unexpected error messages on CEMTClass in TN-W

### ISSUE THREAD

**Issue by JohannaOtt** [INSPIRE-MIF/helpdesk-validator#620]
When validating [this WFS](https://haleconnect.com/ows/services/org.292.6cef2128-a227-4aa7-bc74-0e3797644d51_wfs?SERVICE=WFS&REQUEST=GetCapabilities&VERSION=2.0.0) in the staging instance of the validators I get two errors concerning CEMTClass that I cannot understand when looking at the data:

1. Test ID EID5c4808a6-566d-3b93-a5e5-e6002f4fc8a7 returns: Unexpected result evaluating XPath expression with context node tn-w:CEMTClass:
 ns1:validFrom[@xsi:nil='true'] expected [true] but found [false]

2. Test ID EID6d46ac1b-5490-3b3f-a19a-1487ae31dc22 returns: Unexpected number of {http://inspire.ec.europa.eu/schemas/tn-w/4.0}CEMTClass descendant elements. expected [2] but found [4]

This is an excerpt of the data with all CEMTClass objects contained
[CEMTClass.zip](https://github.com/INSPIRE-MIF/helpdesk-validator/files/7260733/CEMTClass.zip)

**Comment by dperezBM** [comment]
Dear @JohannaOtt , 

Thank you for your message. As we commented on issue #622, we have tested your service directly against [OGC Validator](https://cite.opengeospatial.org/teamengine/) and the results are the same that we have with the validator. 

![image](https://user-images.githubusercontent.com/33548409/136375518-c68172f4-405b-4a7d-a84e-44ae42dd9ad3.png)




Maybe you could report this issue directly to the [OGC GitHub repository](https://github.com/opengeospatial/teamengine/issues). 

Best regards.

**Comment by JohannaOtt** [comment]
I created https://github.com/opengeospatial/teamengine/issues/482 for it

**Comment by JohannaOtt** [comment]
https://github.com/opengeospatial/ets-wfs20/issues/206 only covered parts of it. There is https://github.com/opengeospatial/ets-wfs20/issues/223 now as well

**Comment by fabiovinci** [comment]
Dear @JohannaOtt,
it seems that the relevant OGC issues were closed, so I will close this issue.
Please reopen it if the error persists.

**Comment by JohannaOtt** [comment]
@fabiovinci Just tested the WFS from the original description in the staging instance (production instance seems not to work and is immediately returning an error) and the errors are remaining.
This is the [link to the report](http://staging-inspire-validator.eu-west-1.elasticbeanstalk.com/etf-webapp/test-run/details.html?id=EIDf4cb03d0-9ded-43c4-8bef-e92f0c304e41).
So seems like the changes in the OGC Validator did not solve them. Not sure if that is something that now needs to be addressed in the INSPIRE validator or if it needs to be re-directed to the OGC Validator team.
Unfortunately, I cannot re-open the issue myself.

**Comment by fabiovinci** [comment]
Dear @JohannaOtt,

the INSPIRE Validator should use the latest version of the OGC Validator, we will test the service in production and compare results.

**Comment by fabiovinci** [comment]
Dear @JohannaOtt,

I validated your service with the production instance ([see the report](https://inspire.ec.europa.eu/validator/test-run/details.html?id=EID17f0e121-e53e-445b-a840-66d9502a9842)) and directly with the OGC Validator ([Test Session Results.zip](https://github.com/INSPIRE-MIF/helpdesk-validator/files/12069638/Test.Session.Results.zip)) and the results are the same.

The fix for the OGC issue 223 is assigned to milestone 1.39, but the one included in the [Official OGC Validator](https://cite.opengeospatial.org/teamengine/) is 1.38.

The correct implementation of the fix of your issue could be tested in the [Beta OGC Validator](https://cite.opengeospatial.org/te2/) which implements the 1.40 version of the WFS2.0 specification.

**Comment by JohannaOtt** [comment]
@fabiovinci I agree that the errors are not occurring any longer in the beta version of the OGC validator, so this issue can be closed IMO.

**Comment by fabiovinci** [comment]
Dear all,
since version 1.40 of the WFS 2.0 specification is now included in the production instance, this error should no longer be present.
We will close the issue. Please reopen it if the error persists.

**Comment by JohannaOtt** [comment]
I validated [this WFS](https://haleconnect.com/ows/services/org.292.2cfb8909-9a7c-4564-a4cc-5e86bb3a8c0e_wfs?SERVICE=WFS&REQUEST=GetCapabilities&VERSION=2.0.0) (similar to the one from the original issue description) with the INSPIRE validator (production instance) and can confirm that there are no errors concerning the CEMTClass reported any longer.
